docs/docs/cmd/planner/bucket/bucket-get.md

```diff
@@ -0,0 +1,57 @@
+# planner bucket get
+
+Retrieves the specified Microsoft Planner bucket
+
+## Usage
+
+```sh
+m365 planner bucket get [options]
+```
+
+## Options
+
+`-i, --id [id]`
+: ID of the bucket to retrieve details. Specify either `id` or `title` but not both.
+
+`-t, --title [title]`
+: Title of the bucket to retrieve details. Specify either `id` or `title` but not both. 
+
+`--planId [planId]`
+: Plan ID to which the bucket belongs. Specify either `planId` or `planName` when using `title`.
+
+`--planName [planName]`
+: Plan Name to which the bucket belongs. Specify either `planId` or `planName` when using `title`.
+
+`--ownerGroupId [ownerGroupId]`
+: ID of the group to which the plan belongs. Specify `ownerGroupId` or `ownerGroupName` when using `planName`.
+
+`--ownerGroupName [ownerGroupName]`
+: Name of the group to which the plan belongs. Specify `ownerGroupId` or `ownerGroupName` when using `planName`.
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Gets the specified Microsoft Planner bucket 
+
+```sh
+m365 planner bucket list --id "5h1uuYFk4kKQ0hfoTUkRLpgALtYi"
+```
+
+Gets the Microsoft Planner bucket in the PlanId xqQg5FS2LkCp935s-FIFm2QAFkHM
+
+```sh
+m365 planner bucket list --title "Planner Bucket A" --planId "xqQg5FS2LkCp935s-FIFm2QAFkHM"
+```
+
+Gets the Microsoft Planner bucket in the Plan _My Plan_ owned by group _My Group_
+
+```sh
+m365 planner bucket list --title "Planner Bucket A" --planName "My Plan" --ownerGroupName "My Group"
```

Comment:
`m365 planner bucket list` should be `m365 planner bucket get`.

Reply:
Thanks for spotting the copy error. I have corrected it.